src/types/route.ts

```diff
@@ -10,7 +10,7 @@ export type Routes = Readonly<Route[]>
 /**
  * The Route properties originally provided to `createRoutes`. The only change is normalizing meta to always default to an empty object.
  */
-type RoutePropsWithMeta = RouteProps & { meta: RouteMeta }
+export type RoutePropsWithMeta = RouteProps & { meta: RouteMeta }
```

Comment:
@stackoverfloweth I exported this because we had some test utils that were recreating this type. Seems like we should just use this one. But curious if there's a reason it might have been duplicated before.

Reply:
probably just an attempt to keep a lid on how many "route" types we have available, even if only internally

Reply:
but I agree that duplicating doesn't feel right either

Reply:
Idk that we need to export this publicly. But we can use it internally.